COPYRIGHT-NOTICES.md

```diff
@@ -0,0 +1,138 @@
+  whether a minor contribution (such as a typo fix) means that a new copyright
+  notice should be added.
+- Adding many different copyright notices may increase the burden on downstream
+  distributors, if their license compliance processes involve reproducing
+  notices.
+- The specific individual or legal entity that owns the copyright might not be
+  known to the contributor; it could be you, your employer, or some other entity.
+
+## Copyright notices for website footers
+
+Please use one of these standard footers, which have been approved by OpenJS
+Foundation legal counsel and the Board of Directors:
+
+### For nodejs.org, and projects which reference Node.js
+
+> Copyright [OpenJS Foundation](https://openjsf.org) and \[project
+name\] contributors. All rights reserved. The [OpenJS
+Foundation](https://openjsf.org) has registered trademarks and uses
+trademarks.  For a list of trademarks of the [OpenJS
+Foundation](https://openjsf.org), please see our [Trademark
+Policy](https://trademark-policy.openjsf.org/) and [Trademark
+List](https://trademark-list.openjsf.org/).  Node.js is a trademark of
+Joyent, Inc. and is used with its permission.  Trademarks and logos not
+indicated on the [list of OpenJS Foundation
+trademarks](https://trademark-list.openjsf.org) are trademarks™ or
+registered® trademarks of their respective holders. Use of them does not
+imply any affiliation with or endorsement by them.
+> 
+> [The OpenJS Foundation](https://openjsf.org/) \| [Terms of
+Use](https://terms-of-use.openjsf.org/) \| [Privacy
+Policy](https://privacy-policy.openjsf.org/) \| [OpenJS Foundation
+Bylaws](https://bylaws.openjsf.org/) \| [Trademark
+Policy](https://trademark-policy.openjsf.org/) \| [Trademark
+List](https://trademark-list.openjsf.org/) \| [Cookie
+Policy](https://www.linuxfoundation.org/cookies/)
+
+### For projects unrelated to Node.js:
+
+> Copyright [OpenJS Foundation](https://openjsf.org) and \[project
+name\] contributors. All rights reserved. The [OpenJS
+Foundation](https://openjsf.org) has registered trademarks and uses
+trademarks.  For a list of trademarks of the [OpenJS
+Foundation](https://openjsf.org), please see our [Trademark
+Policy](https://trademark-policy.openjsf.org/) and [Trademark
+List](https://trademark-list.openjsf.org/).  Trademarks and logos not
+indicated on the [list of OpenJS Foundation
+trademarks](https://trademark-list.openjsf.org) are trademarks™ or
+registered® trademarks of their respective holders. Use of them does not
+imply any affiliation with or endorsement by them.
+> 
+> [The OpenJS Foundation](https://openjsf.org/) \| [Terms of
+Use](https://terms-of-use.openjsf.org/) \| [Privacy
+Policy](https://privacy-policy.openjsf.org/) \| [OpenJS Foundation
+Bylaws](https://bylaws.openjsf.org/) \| [Trademark
+Policy](https://trademark-policy.openjsf.org/) \| [Trademark
+List](https://trademark-list.openjsf.org/) \| [Cookie
+Policy](https://www.linuxfoundation.org/cookies/)
```

Comment:
Please reformat these as fenced code blocks (without backslash escapes); it's difficult to copy-paste it.  If you try to copy the raw code, you get escapes, but if you try to copy the rendered text, you lose the links.

Reply:
I just pushed a patch which adds rendered HTML, markdown in code fences and without line breaks or escaped chars, and raw HTML.